### PR TITLE
add wallace IT

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: tools/interactive/interactivetool_wallace.xml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<tool id="interactive_tool_wallace" tool_type="interactive" name="Wallace" version="0.1">
+    <description>Webbased Interactive modeling of species niches and distributions</description>
+    <requirements>
+        <container type="docker">quay.io/bgruening/wilson-app</container>
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml]: @yvanlebras we need the container here.

File: tools/interactive/interactivetool_wallace.xml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            #if $__user__:
+                #for $api_key in $__user__.api_keys:
+                    ${api_key.key}
+                    #break
+                #end for
+            #end if
+        </environment_variable> <!-- FIXME: We should have a better way to get user's API key -->
+    </environment_variables>
+    <command><![CDATA[
+        ln -s '${infile}' /srv/shiny-server/data/inputdata.txt &&
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml:27]: Does this look ok for you?

File: tools/interactive/interactivetool_wallace.xml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        /usr/bin/shiny-server.sh
+    ]]>
+    </command>
+    <inputs>
+        <param name="infile" type="data" format="tabular,txt" label="Tabular in Clarion format"/>
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml:32]: label needs to be fixed, what kind of format is this?

File: tools/interactive/interactivetool_wallace.xml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    </inputs>
+    <outputs>
+        <data name="outfile" format="txt" />
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml]: outfile can be removed I think ...

File: tools/interactive/interactivetool_wallace.xml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    </outputs>
+    <tests>
+    </tests>
+    <help>
+<![CDATA[
+
+`Wallace <https://github.molgen.mpg.de/loosolab/wilson-apps/wiki>`_ is a modular platform for reproducible modeling of species niches and distributions.
```
Comment (bgruening) [inline tools/interactive/interactivetool_wallace.xml:45]: @yvanlebras please feel free to add more help ... e.g. the input format.

File: tools/interactive/interactivetool_wallace.xml (hunk 1 above)
Comment (yvanlebras) [reply tools/interactive/interactivetool_wallace.xml]: you made it (If I'm not wrong) (how that I am too slow ;) )

**Pull request merged**